circle.yml

```diff
@@ -1,10 +1,15 @@
 machine:
   xcode:
-    version: 7.3
+    version: 8.2
 dependencies:
   pre:
     - brew update || brew update
     - brew install swiftlint
+    - system_profiler SPSoftwareDataType
+    - security list-keychains
+    - security find-identity -p codesigning
+    - instruments -s devices
+    - xcodebuild -showsdks
```

Comment:
this is just a bunch of diagnostic stuff i copied from the main repo. can be handy to get this info about the circleci boxes running the tests.